### PR TITLE
UnneededControlParenthesesFixer - add "break" and "continue" support

### PR DESCRIPTION
**Pull request by gharlan**

### CODE REVIEW

File: Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php [hunk 1]
```diff
@@ -62,6 +74,34 @@ public function provideFixCases()
     {
         return array(
             array(
+                '<?php break;',
+            ),
+            array(
+                '<?php break 2;',
```
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php]: can be removed, covered in next test case.

File: Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php [hunk 2]
```diff
@@ -62,6 +74,34 @@ public function provideFixCases()
...
+                '<?php break(2);',
+            ),
+            array(
+                '<?php continue;',
+            ),
+            array(
+                '<?php continue 2;',
```
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php]: test in the next case as well :)

File: Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php (hunk 1 above)
Comment (gharlan) [inline Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php]: thanks, both are fixed. (but github moved your comments to wrong lines..)

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php]: thanks you :+1: :)

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/UnneededControlParenthesesFixerTest.php]: thank you :)

**Comment by SpacePossum** [comment]
lgtm :+1:

File: Symfony/CS/Fixer/Symfony/UnneededControlParenthesesFixer.php [hunk 3]
```diff
@@ -26,18 +27,22 @@
...
 
     private static $loops = array(
-        'switch_case' => array('lookupTokens' => T_CASE, 'neededSuccessors' => array(';', ':')),
+        'break' => array('lookupTokens' => T_BREAK, 'neededSuccessors' => array(';')),
+        'continue' => array('lookupTokens' => T_CONTINUE, 'neededSuccessors' => array(';')),
+        'clone' => array('lookupTokens' => T_CLONE, 'neededSuccessors' => array(';', ':', ',', ')')),
         'echo_print' => array('lookupTokens' => array(T_ECHO, T_PRINT), 'neededSuccessors' => array(';', array(T_CLOSE_TAG))),
         'return' => array('lookupTokens' => T_RETURN, 'neededSuccessors' => array(';')),
-        'clone' => array('lookupTokens' => T_CLONE, 'neededSuccessors' => array(';', ':', ',', ')')),
+        'switch_case' => array('lookupTokens' => T_CASE, 'neededSuccessors' => array(';', ':')),
```
Comment (Soullivaneuh) [inline Symfony/CS/Fixer/Symfony/UnneededControlParenthesesFixer.php]: Why changing placement?

Comment (gharlan) [inline Symfony/CS/Fixer/Symfony/UnneededControlParenthesesFixer.php]: alphabetical order ;)

**Comment by gharlan** [comment]
> keradus changed the title from **UnneededControlParenthesesFixer - add `break` and `continue`** to **UnneededControlParenthesesFixer - add "break" and "continue"** 22 hours ago

haha, tried to fit your naming conventions, but failed again. ;)

**Comment by keradus** [comment]
I would use >`< on my own too, but >`< in `.md` files is a tricky char that need to be escaped :/
And since changelog file is generated from title - I simplify it to not have need of escaping.

**Comment by keradus** [comment]
Thank you @gharlan.

**Pull request merged**